db_functions.py

```diff
@@ -121,75 +126,75 @@ def registrace(jmeno, prijmeni, ulice, mesto, psc, email, telefon, heslo, heslo_
     finally:
         if conn is not None:
             conn.close()
-    return id_uzivatele
+    return id_user
 
 
-def najdi_uzivatele(email):
-    """ najde uzivatele v databazi """
+def find_user(email):
+    """ Finds user in the database. """
 
     sql = """SELECT id_uzivatele, jmeno, prijmeni, email, heslo, telefon FROM uzivatele WHERE lower(email)=%s;"""
     conn = get_db()
 
-    uzivatel = None
+    user = None
     try:
         cur = conn.cursor()
         cur.execute(sql, (email.lower(),))
         # get the generated id back
-        uzivatel = cur.fetchone()
+        user = cur.fetchone()
         # close communication with the database
         cur.close()
     except (Exception, psycopg2.DatabaseError) as error:
         print(error)
     finally:
         if conn is not None:
             conn.close()
-    if uzivatel:
-        return User(uzivatel[3], uzivatel[0], uzivatel[4], uzivatel[1], uzivatel[2], uzivatel[5])
+    if user:
+        return User(user[3], user[0], user[4], user[1], user[2], user[5])
     else:
         return None
 
 
+def car_exists(driver, id_race):
+    """ Looks into database if the driver is not added for given race. """
 
-def uz_existuje_auto(ridic, id_zavod):
-    """ podiva se do db, jestli uz tam neni ridic pro dany zavod """
-
-    sql_najdi_auto = """SELECT * from nabidka_spolujizdy WHERE ridic = %s and id_zavod = %s;"""
+    sql_find_car = """SELECT * from nabidka_spolujizdy WHERE ridic = %s and id_zavod = %s;"""
 
     conn = get_db()
     
     try:
         cur = conn.cursor()
-        cur.execute(sql_najdi_auto, (ridic, id_zavod))
+        cur.execute(sql_find_car, (driver, id_race))
         # execute the SELECT statement
-        vysledek = cur.fetchall()
+        result = cur.fetchall()
         cur.close()
     finally:
         if conn is not None:
             conn.close()
-    return vysledek
+    return result
 
-def nove_auto(ridic, id_zavod, misto_odjezdu, datum_odjezdu, mist_auto_nabidka, poznamky):
-    """ prida nove auto s ridicem do databaze """
+
+def add_carpooling_offer(driver, id_race, departure, departure_date, offer_of_places_in_car, notes):
+    """ Adding a new car with a driver into database. """
 
     sql_najdi_auto = """SELECT * from nabidka_spolujizdy WHERE ridic = %s and id_zavod = %s;"""
 
     sql_zapis_auto = """INSERT INTO nabidka_spolujizdy(ridic, id_zavod, misto_odjezdu, datum_odjezdu, mist_auto_nabidka, poznamky)
              VALUES(%s, %s, %s, %s, %s, %s) RETURNING id_jizdy; """
 
     conn = get_db()
-    id_jizdy = None
+    id_race = None
```

Comment:
This should be `id_ride` instead of `id_race`, because `id_race` is an argument of this function, while `id_ride` is the ID of the newly created carpooling offer.